libs/route-config/README.md

```diff
@@ -105,11 +107,62 @@ Now we can use it in the component's template
   This text is only visible, if there is a 'show' tag in the route data's `routeTags` Array
 </p>
 <ng-template #noShowTag>
-  <p>There is no show tag in this route's config</p>
+  <p>There is no 'show' tag in this route's config</p>
+</ng-template>
+```
+
+---
+
+#### RouteDataHasDirective (`*tdRouteDataHas`)
+
+In case you need to use a different route data's property to store the tags you can use `*tdRouteDataHas` directive. It works very similarly to `*tdRouteTag` directive but provides a way to use different property as a source of data.
```

Comment:
`If you need to use a different route data property to store the tags, you can use the `*tdRouteDataHas` directive.` <-- you don't need the possessive `'s` there, don't start the sentence with `In case` and add a comma after `tags`

Reply:
`similarly` --> `similar`

`different property` --> `different properties`

Reply:
```suggestion
If you need to use a different route data property to store the tags, you can use `*tdRouteDataHas` directive. It works very similar to `*tdRouteTag` directive but provides a way to use different properties as a source of data.
```

Reply:
this is resolved by the next suggestion